app/scripts/lib/experiments/choices/TEMPLATE.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Template file for new rules. To use:
+ *
+ * 1. Copy TEMPLATE.js to a new rule file.
+ * 2. Change `ChangeMeExperiment` class name to another name.
+ * 3. Fill in an experiment name for `CHANGE_ME` in the constructor.
+ * 4. Fill in the `choose` function.
+ * 5. Include the new rule file in index.js.
+ * 6. Access in views via `this.experiments.choose('experiment name from 3')`
+ */
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseExperiment = require('./base');
+
+  module.exports = class ChangeMeExperiment extends BaseExperiment {
```

Comment:
nit: `ChangeMeExperiment` should probably be called `ChangeMeExperimentChoice` or `ChangeMeExperimentChoiceRule`